index.html

```diff
@@ -213,7 +220,7 @@
           </li>
         </ol>
       </li>
-      <li>For each <i>observer</i> in <i>interested observers</i>:
+      <li>If the [load event] of the [current document] is not yet completed, for each <i>observer</i> in <i>interested observers</i>:
```

Comment:
Hmm, wouldn't this mean that we're not queueing any observer events until document is completely loaded? 

- I should be able to register an observer before onload event and observe all events after it
  - Optionally, I should be able to pass in a flag indicating that I also want this observer to consume all buffered events of the specified types. 

Conceptually, I think what we want to do here is:
- Keep existing queue logic as is
- Add an extra step that...
  - If document is not [completely loaded](https://html.spec.whatwg.org/#completely-loaded), append the event to the global performance timeline. 
- As part of the register observer algorithm, check for the boolean for buffered events
  - If the boolean is there, retrieve all the events of specified type and append them to the observer buffer